### ISSUE TITLE
Commented out stat

### ISSUE THREAD

**Issue by nathanjordan** [m3db/m3#2313]
I notice the `messageConsumeLatency` timer in m3msg was commented out in #2230 . Was this intentional? We are planning on alerting on this metric.

```go
func (w *messageWriterImpl) Ack(meta metadata) bool {
	// acked, initNanos := w.acks.ack(meta)
	acked, _ := w.acks.ack(meta)
	if acked {
		// w.m.messageConsumeLatency.Record(time.Duration(w.nowFn().UnixNano() - initNanos))
		w.m.messageAcked.Inc(1)
		return true
	}
	return false
}
```

https://github.com/m3db/m3/blob/599722254d0c9df2416e2904213db33e6e572055/src/msg/producer/writer/message_writer.go#L309

**Comment by nathanjordan** [comment]
@robskillington

**Comment by cw9** [comment]
https://github.com/m3db/m3/pulls

**Comment by gibbscullen** [comment]
Fix merged with #2372
